### PR TITLE
MGMT-4048 change disk id to be the full path of the by-id link

### PR DESCRIPTION
**Pull request by michaellevy101**

### CODE REVIEW

File: src/inventory/disks.go [hunk 1]
```diff
@@ -43,18 +43,18 @@ func (d *disks) getDisksWWNs() map[string]string {
 	// For example: wwn-0x6141877064533b0020adf3bc0325d664	-> /dev/sdb
 	// "wwn-0x6141877064533b0020adf3bc0325d664" is the disk id and the path is: /dev/sdb
 	return funk.Map(matchingFiles, func(fileInfo os.FileInfo) (string, string) {
-		diskId := fileInfo.Name()
-		diskPath, err := d.dependencies.EvalSymlinks(filepath.Join("/dev/disk/by-id", diskId))
+		diskId := filepath.Join("/dev/disk/by-id", fileInfo.Name())
```
Comment (omertuc) [inline src/inventory/disks.go:46]: Lines 25 and 46 duplicate this path, it should be in a var

File: src/inventory/disks.go [hunk 2]
```diff
@@ -68,7 +68,7 @@ func (d *disks) getPath(busPath string, diskName string) string {
 	if err == nil {
```
Comment (omertuc) [inline src/inventory/disks.go:68]: GitHub won't allow me to comment but the first line of this function should be filepath.Join rather than Sprintf

Comment (michaellevy101) [reply src/inventory/disks.go:69]: because i didn't change this method.... but i changed it

File: src/inventory/disks.go (hunk 1 above)
Comment (michaellevy101) [reply src/inventory/disks.go]: done

**Pull request merged**